src/libraries/reduxUtils/convert.ts

```diff
@@ -0,0 +1,15 @@
+import { format } from "date-fns";
+import { SummaryFieldType } from "./SummaryFieldType";
+
+export const convertToSummaryData = (
```

Comment:
We could name the file with the very same name of the function inside it.

Reply:
Ok, in that case the file just belongs to that function. I name like that because we could add another function inside.